app/lib/api_support/params.rb

```diff
@@ -10,7 +10,7 @@ module ApiSupport::Params
     private
 
     def flat_params
-      params.except(:format, :controller, :action, :provider_key, :api_key)
+      params.except(:format, :controller, :action, :provider_key, :api_key, :access_token)
```

Comment:
could you explain this change?

Reply:
Yes, `flat_params` goal is to return the keys that actually mater, that is, the parameters belonging to the resource we are working with. It returns all the keys except those that are usually received in requests but we don't care about. I added the `:access_token` to this list, that way it will be always ignored by `ForbidParams`.